draft-ietf-quic-transport.md

```diff
@@ -2841,23 +2841,24 @@ needing acknowledgement are received.  The sender can use the receiver's
 Strategies and implications of the frequency of generating acknowledgments are
 discussed in more detail in {{QUIC-RECOVERY}}.
 
-To limit ACK Blocks to those that have not yet been received by the sender, the
-receiver SHOULD track which ACK frames have been acknowledged by its peer.  Once
-an ACK frame has been acknowledged, the packets it acknowledges SHOULD NOT be
-acknowledged again.
+To limit the ranges of acknowledged packet numbers to those that have not yet
+been received by the sender, the receiver SHOULD track which ACK frames have
+been acknowledged by its peer.  Once an ACK frame has been acknowledged, the
+packets it acknowledges SHOULD NOT be acknowledged again.
```

Comment:
FWIW, I agree.  #2199

Reply:
How about: ... _Once an ACK frame has been acknowledged, it is RECOMMENDED to not repeat acknowledgement of the same packets._ The point is, there is no harm whatsoever in repeating the range multiple times, but it is good practice to keep ACK frames short.

Reply:
+1 to Christian's suggestion or something along those lines.

Reply:
... Once an ACK frame has been acknowledged, it is RECOMMENDED to not repeat acknowledgement of the same packets whenever this can reduce the overall size of the next ACK frame.